nodejs/week1/homework/warmup.js

```diff
@@ -0,0 +1,18 @@
+console.log("inside warmup file");
+
+class Circle {
+    constructor(radius) {
+        this.radius = radius;
+    }
+    getDiameter() { return this.radius * 2 }
+    getCircumference() { return this.getDiameter() * Math.PI }
+    getArea() { return Math.PI * this.radius * this.radius }
```

Comment:
Good job, but try to use exponents(power) to avoid repetition of `radius`.  There is a function you can use in `Math` for that.